man/ostree-prepare-root.xml

```diff
@@ -113,6 +113,14 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
                 <term><varname>sysroot.readonly</varname></term>
                 <listitem><para>A boolean value; the default is false.  If this is set to <literal>true</literal>, then the <literal>/sysroot</literal> mount point is mounted read-only.</para></listitem>
           </varlistentry>
+            <varlistentry>
+                <term><varname>composefs.enable</varname></term>
+                <listitem><para>This can be yes, no or maybe, the default is false.  If this is set to <literal>yes</literal>, then composefs is always used, and boot fail if it is not available. If set to <literal>maybe</literal>, then composefs is used if supported.</para></listitem>
```

Comment:
```suggestion
                <listitem><para>This can be yes, no or maybe, the default is no.  If this is set to <literal>yes</literal>, then composefs is always used, and the boot fails if it is not available. If set to <literal>maybe</literal>, then composefs is used if supported.</para></listitem>
```